net/openappid/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openappid
+PKG_VERSION:=26425
+PKG_RELEASE:=1
+
+PKG_SOURCE:=snort-$(PKG_NAME).tar.gz
```

Comment:
This looks odd, makes me wonder, where is used PKG_VERSION.

Reply:
Yes, upstream does not version their tarball so it is odd.  That is in the download URL.
```
https://www.snort.org/downloads/openappid/26425
```